### PR TITLE
fix : fixing path user json

### PR DESCRIPTION
**Pull request by CatCoder36**
Description: I have fixed the json user path 
Reviewer: @zrock-dev

### CODE REVIEW

**Review by zrock-dev — CHANGES_REQUESTED** [review]
For the tests in BTreeRepositoryTest use the pathUserDataBase constant.

Hint: 
```java
String.format("%s/TestNode1.json", BTreeRepository.PATH_USER_DATA_BASE) 
```

**Comment by zrock-dev** [comment]
fixes #20

File: backend/src/main/java/com/fake_orgasm/users_management/repository/BTreeRepository.java [hunk 1]
```diff
@@ -22,7 +24,7 @@ public class BTreeRepository implements IBTreeRepository<User> {
     /**
      * Path where the nodes are saved.
      */
-    private final String pathUserDataBase = "../backend/src/main/resources/DataBase/Users";
+    private final String pathUserDataBase = "../backend/user_database";
```
[the remark this answers is not in the document]
Comment (CatCoder36) [reply backend/src/main/java/com/fake_orgasm/users_management/repository/BTreeRepository.java:27]: The check style does not allow such names with uppercase only.

[the remark this answers is not in the document]
Comment (zrock-dev) [reply backend/src/main/java/com/fake_orgasm/users_management/repository/BTreeRepository.java:27]: it does, as an example you can check the file: src/main/java/com/fake_orgasm/generator/user_generator/UserNameGenerator.java

[the remark this answers is not in the document]
Comment (CatCoder36) [reply backend/src/main/java/com/fake_orgasm/users_management/repository/BTreeRepository.java]: solved

**Review by zrock-dev — COMMENTED** [review]
Good job

**Review by zrock-dev — APPROVED** [review]

**Pull request merged**